Add Winter 2023 Fanart Contest submissions news post (#10411)
ppy/osu-wiki: news/2023/2023-11-07-winter-fanart-contest.md

---
layout: post
title: "Winter 2023 Fanart Contest"
date: 2023-11-07 08:00:00 +0000
---

The end of the year soon approaches, and what better way to send it off in style than with a feast-themed fanart contest!

![](https://assets.ppy.sh/contests/189/header.jpg)

> Each and every year since a date long-lost to the annals of time, we've sat down and posted the artists of the osu! community a timeless question: **do you like to draw?**

Reviewer: ```suggestion
Each and every year since a date long lost to the annals of time, we've sat down and posed the artists of the osu! community a timeless question: **do you like to draw?**
```

Author: libtex disagrees with you firmly re long-lost

Reviewer: libtex is wrong because "long lost" isn't used to modify a noun, where it would be hyphenated

google "long lost to time". no one hyphenates "long lost" in this case

> If your answer to this question is yes, then pull up a chair and prepare yourselves for a brand new food-related theme! **Draw any of the osu! mascots (or community mascots) enjoying a seasonal feast** for fun and glory!

Reviewer: ```suggestion
If your answer to this question is yes, then pull up a chair and prepare yourselves for a brand-new food-related theme! **Draw any of the osu! mascots (or community mascots) enjoying a seasonal feast** for fun and glory!
```

You may be wondering what the heck a *seasonal feast* is. Since we're a massively multicultural community, there's a ton of traditional festivals and feasts that roughly align around the end of the year, from anything like Thanksgiving, Diwali/Deepavali, Hanukkah, the winter solstice (including the Dongzhi festival), Festivus, Christmas, Kwanzaa, and tons more that we could probably spend a whole article describing.

> Suffice to say, the end of the year involves a *lot* of celebrating and eating, so we figured this year, we'd try trussing up the idea and seeing what delectable creations the immensely talented artists of the osu! community cook up. Shokugeki no Soma enthusiasts who've wanted to draw glazed ham (and we know there is at LEAST *one* of you), rejoice!

Reviewer: ```suggestion
Suffice to say, the end of the year involves a *lot* of celebrating and eating, so we figured this year, we'd try trussing up the idea and seeing what delectable creations the immensely talented artists of the osu! community can cook up. *Shokugeki no Soma* enthusiasts who've wanted to draw glazed ham (and we know there is at LEAST *one* of you), rejoice!
```

If you're looking for ideas, here's a few to get the juices flowing:

- The mascots fighting over who gets the last piece of Christmas ham
- Mocha rolling a comically overstuffed don around in front of a fireplace

> - Yuzu being spear-tackled by Chirou right as he was about to finish his fruit pyramid centerpiece
> - Mari pulverizing Mani with a turkey drumstick for the sin of being an annoying younger sibling

Reviewer: ```suggestion
- Yuzu being spear-tackled by Chirou right as he was about to finish his fruit pyramid centrepiece
- Mari pulverising Mani with a turkey drumstick for the sin of being an annoying younger sibling
```

- ...and basically anything you can think of that involves a seasonal, end-of-year feast!

> This is a pretty broad theme, so feel free to go wild with it - just make sure it stays vaguely festive. If you want snow, go for it! If you don't, that's fine too. As long as the mascots are enjoying their time together in the company of good friends, we'll consider that fair game.

Reviewer: ```suggestion
This is a pretty broad theme, so feel free to go wild with it — just make sure it stays vaguely festive. If you want snow, go for it! If you don't, that's fine too. As long as the mascots are enjoying their time together in the company of good friends, we'll consider that fair game.
```

You've got **21 days** to bring your creation to life, so time to get cracking on the crackling! As always, a selection of the top voted finalists will make it to the main menu and receive 3 months of osu!supporter, plus the top 3 will also receive fancy bespoke profile badges pulled from their own creations.

But make sure you read the contest rules below first, especially because there's a new one this time around.

## Rules

- **By submitting your artwork to this contest, you grant us permission to use your artwork in the main menu client should it win, and to use it in promotional material for both the game (osu!) and the contest (Winter 2023 Fanart Contest).** You retain all rights to your creation and you can sell it and distribute it however you like — it is your work, after all. We're just saying here that if you submit your piece to this contest, you should be okay with us using it.

> - **All entries MUST follow the contest theme (Winter/end-of-year seasonal feast) and contain AT LEAST one of the osu! official or community mascots.** The mascots are not optional and neither is the feast/festival/celebration!

Reviewer 2: `feast/festival/celebration` makes it sounds as if food does not actually need to be involved, but from reading the newspost it seems to be the actual focus?

Reviewer: considering the amount this is emphasised in the post, with the prompt being "Draw any of the osu! mascots (or community mascots) enjoying a seasonal feast" and the theme being "Winter/end-of-year seasonal feast", i don't think there's a whole lot of room for misunderstanding

Reviewer: ```suggestion
- **All entries MUST follow the contest theme (Winter/end-of-year seasonal feast) and contain AT LEAST one of the osu! official or community mascots.** The mascots are not optional and neither are the feasts and festivities!
```

- **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** The circular spot in the middle shows where the main-menu cookie will be when inside the game. Factor this in when designing your piece, as anything inside it is likely not going to be seen. This doesn't mean leaving a big circular hole full of nothing in the middle! Be creative. *Make sure you don't leave it visible in your end result.*
- **Do not leave the dotted osu! cookie template visible in your submission.** Leave the meme in the grave, where it belongs.

> - **If you're using characters from other intellectual properties, do so sparingly.** All the official osu! mascots and the community mascots are fair game, but please avoid using excessive representations of characters from other worlds or franchises in your work - the contest is about osu! foremost, after all. Remember also that the community mascots are someone else's treasured creation, so please treat them with the same respect that you'd treat your own OCs and work.

Reviewer: ```suggestion
- **If you're using characters from other intellectual properties, do so sparingly.** All the official osu! mascots and the community mascots are fair game, but please avoid using excessive representations of characters from other worlds or franchises in your work — the contest is about osu! foremost, after all. Remember also that the community mascots are someone else's treasured creation, so please treat them with the same respect that you'd treat your own OCs and work.
```

The [Winter 2023 Fanart Contest listing](https://osu.ppy.sh/community/contests/189) awaits you when you're ready to submit your entry.

And remember, you can resubmit your entry as many times as you like before submissions close. One pixel off? No worries, fix the thing and reupload your entry. Do this thirty times, three hundred times, three THOUSAND times, it's fine! Though if you're doing it that many times, we might wonder just how fine you are, but we digress.

Good luck!

> —Ephemeral

Reviewer 2: Not sure how past newsposts handled this, but:
The contest listing has the header attribution after the rules, perhaps it should also be here? Credit where credit is due

Author: good idea, added

- **Do not leave the dotted osu! cookie template visible in your submission.** Leave the meme in the grave, where it belongs.
- **If you're using characters from other intellectual properties, do so sparingly.** All the official osu! mascots and the community mascots are fair game, but please avoid using excessive representations of characters from other worlds or franchises in your work — the contest is about osu! foremost, after all. Remember also that the community mascots are someone else's treasured creation, so please treat them with the same respect that you'd treat your own OCs and work.

> *Header & promotional artwork by finalist [Novarece](https://osu.ppy.sh/users/18728833) from the [New Beginnings Art Contest](https://osu.ppy.sh/community/contests/160)!*

Reviewer: i generally disagree with ampersand other than in headings or titles (it's random to use it once here)

```suggestion
*Header and promotional artwork by finalist [Novarece](https://osu.ppy.sh/users/18728833) from the [New Beginnings Art Contest](https://osu.ppy.sh/community/contests/160)!*
```
